bin/assets/scripts/loc.js

```diff
@@ -43,3 +48,25 @@ function handleHash() {
     }
   }
 }
+
+if (token) {
+  deleteButton.addEventListener('click', () => {
+    fetch(window.location.pathname, {
+      method: 'DELETE',
+      headers: {
+        Authorization: `Token ${token}`
+      }
+    })
+      .then((res) => {
+        if (!res.ok) {
+          console.error(res);
+          throw new Error(res.statusText);
+        }
+        window.location.href = '/';
+      })
+      .catch((err) => {
+        alert(`Une erreur est survenue (${err.message}), votre snippet n'a donc pas pu être supprimé. Regardez la console pour plus d'informations.`);
+        console.error("Nous vous recommendons d'ouvrir une issue sur https://github.com/readthedocs-fr/bin-server si le problème persiste.", err);
```

Comment:
Une petite typo s'est glissée ici
```suggestion
        console.error("Nous vous recommandons d'ouvrir une issue sur https://github.com/readthedocs-fr/bin-server si le problème persiste.", err);
```